test/core/end2end/fixtures/proxy.cc

```diff
@@ -81,7 +81,15 @@ grpc_end2end_proxy* grpc_end2end_proxy_create(const grpc_end2end_proxy_def* def,
 
   grpc_end2end_proxy* proxy =
       static_cast<grpc_end2end_proxy*>(gpr_malloc(sizeof(*proxy)));
-  memset(proxy, 0, sizeof(*proxy));
+  proxy->proxy_port = nullptr;
```

Comment:
Similar comment here: I don't see any reason not to move this more toward C++ syntax.